pageserver/src/http/routes.rs

```diff
@@ -494,7 +494,8 @@ async fn timeline_delete_handler(
         .instrument(info_span!("timeline_delete", tenant = %tenant_id, timeline = %timeline_id))
         .await?;
 
-    json_response(StatusCode::OK, ())
+    // FIXME: needs to be an error for console to retry it. Ideally Accepted should be used and retried until 404.
```

Comment:
Is this FIXME still up to date?

Reply:
Ok, sounds good to me.

Reply:
Decided to go ahead and PR it https://github.com/neondatabase/cloud/pull/5320

Reply:
Merged